CHANGELOG.md

```diff
@@ -15,10 +16,69 @@ For reference, the possible headings are:
 * **Notes**
 
 
+## [0.8.0] - 2016-11-??
+Tag name: v0.8.0
+= commit:
+committed:
+
+### Added
+- The big new thing in version 0.8.0 is support for divisible assets, i.e. assets like carrots or thumbtacks, where the initial CREATE transaction can register/create some amount (e.g. 542 carrots), the first TRANSFER transaction can split that amount across multiple owners, and so on. [Pull Request #794](https://github.com/bigchaindb/bigchaindb/pull/794)
+- Wrote a formal schema for the JSON structure of transactions. Transactions are now checked against that schema. [Pull Request #798](https://github.com/bigchaindb/bigchaindb/pull/798)
+
+### Changed
+- CREATE transactions must now be signed by all `owners_before` (rather than by a federation node). [Pull Request #794](https://github.com/bigchaindb/bigchaindb/pull/794)
+- The user-provided timestamp was removed from the transaction data model (schema). [Pull Request #817](https://github.com/bigchaindb/bigchaindb/pull/817)
+- `get_transaction()` will now return a transaction from the backlog, even if there are copies of the transaction in invalid blocks. [Pull Request #793](https://github.com/bigchaindb/bigchaindb/pull/793)
+- Several pull requests to translate RethinkDB database calls into abstract/generic database calls, and to implement those abstract calls for two backends: RethinkDB and MongoDB. Pull Requests
```

Comment:
Wording seems off (code is translating?), sounds like the code is performing translation. How about:

> Several pull requests to introduce a generalized database interface, to move RethinkDB calls into a separate implementation of that interface, and work on a new implementation for a MongoDB backend.

Could also leave as is.